mrp/setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name="mrp",
-    version="0.1.7",
+    version="0.1.8",
```

Comment:
Side note: Curious if there's a better way to do automatic semantic versioning instead of manually increasing the version number. Probably not worth the effort now, but maybe we can parse previous commits to `fairo/main` and count how many commits that contain modifications to MRP exist before the previous major version.